lang/fr.js

```diff
@@ -151,7 +169,7 @@ var LANGUAGE = {
       },
       "command-files": {
         "title": "Charger/Enregistrer les fichiers",
-        "label": "Charger/Enregistrer"
+        "label": "Ouvrir/Enregistrer"
```

Comment:
We are using the same word in English:
https://github.com/bbcmicrobit/PythonEditor/blob/7cb659ba76bca11b0c26a8cbc0b33ad25eac2687/lang/en.js#L170-L173

It's fine for this merge, but might be worth asking the translation community which one it should be?